src/features/orientation/layouts/orientation/orientation.layout.html

```diff
@@ -53,7 +53,7 @@ <h2 class="h5 text-center text-secondary fw-bold my-3 mt-auto">
                 <div class="w-100 mb-auto position-relative">
                   <img
                     [src]="assetsConfiguration.path + '/img/orientation/' + brandConfiguration.illustration"
-                    alt=""
+                    alt="Carte d'illustration du territoire de la cartographie {{brandConfiguration.name}}"
```

Comment:
```suggestion
                    alt=""
```
Inutile de décrire une illustration si c'est décoratif et n'ajoute pas d'information utile